server/apps/microservices/src/processors/asset-uploaded.processor.ts

```diff
@@ -24,30 +24,17 @@ export class AssetUploadedProcessor {
   /**
    * Post processing uploaded asset to perform the following function if missing
    * 1. Generate JPEG Thumbnail
-   * 2. Generate Webp Thumbnail <-> if JPEG thumbnail exist
+   * 2. Generate Webp Thumbnail
    * 3. EXIF extractor
    * 4. Reverse Geocoding
    *
    * @param job asset-uploaded
    */
   @Process('asset-uploaded')
   async processUploadedVideo(job: Job) {
-    const {
-      asset,
-      fileName,
-      fileSize,
-      hasThumbnail,
-    }: { asset: AssetEntity; fileName: string; fileSize: number; hasThumbnail: boolean } = job.data;
+    const { asset, fileName, fileSize }: { asset: AssetEntity; fileName: string; fileSize: number } = job.data;
```

Comment:
Would be better to use the generic type in the job argument, and if we created the shared interface, here is were we would use it:
```typescript
processUploadedVideo(job: Job<IAssetUploadedJob>) 
```
and then this just becomes:
```typescript
const { asset, fileName, fileSize } = job.data;
```